extension/modules/personas.js

```diff
@@ -1123,8 +1123,8 @@ var PersonaController = {
         // Update the Custom menu. Custom personas unavailable in per-window
         // personas mode.
         let customMenu = document.getElementById("personas-plus-custom-menu");
-        customMenu.hidden = this._prefs.get("perwindow") || !this._prefs.get("showCustomMenu");
-        if (!customMenu.hidden) {
+        customMenu.disabled = this._prefs.get("perwindow")
```

Comment:
Hiding content is generally considered bad user experience. It's better to disable them if they are not available.